routes/web.php

```diff
@@ -19,6 +19,10 @@
     Route::get('faset', function () {
         return view('faset');
     });
+
+    Route::get('faset/admin', function () {
```

Comment:
Considering changing the route to `/admin/faset`

If we put all of these "admin" pages behind admin, we can use a route group to apply consistent authorization.
@kberzinch